mlpp_lib/probabilistic_layers.py

```diff
@@ -350,52 +350,71 @@ def __init__(self, normal):
                         validate_args=validate_args,
                         allow_nan_stats=True,
                     )
+                    self.clip_low = -0.05
+                    self.clip_high = 1.05
```

Comment:
why not using the canonical (0,1) bounds as defaults, while leaving the option to fine tuning them for specific applications like in your case?